modules/apps/content-dashboard/content-dashboard-web/src/main/resources/META-INF/resources/js/components/AuditBarChart.js

```diff
@@ -81,13 +99,76 @@ export default function AuditBarChart({rtl, vocabularies}) {
 		? [BAR_CHART.dotRadiusMin]
 		: [BAR_CHART.legendHeight + BAR_CHART.dotRadiusMin];
 
+	const [checkboxes, setCheckbox] = useState(legendCheckboxes);
+
+	useEffect(() => {
+		if (Object.keys(colors).length) {
+			const head =
+				document.head || document.getElementsByTagName('head')[0];
+			const style = document.createElement('style');
+			style.id = 'legend-checkboxes';
```

Comment:
Is it possible that there will be multiple `AuditBarChart` components on the page? If so, we run the risk of having duplicate ids on the same page by hard coding this id here.

Reply:
Are you sure? What about tomorrow? 

This feels a little like our other conversation last week regarding the mocked data... consider the cost of some fixes and their future risk. Sometimes a fix is simple enough and can save you enough headaches in the future (when you or someone else wants to use this component in a different way than you're expecting) that it's worth the extra time.